terratest/test/helper.go

```diff
@@ -30,7 +31,40 @@ const (
 	DefaultTimeout time.Duration = 5 * time.Second
 )
 
-func queryDNS(dnsServer string, dnsPort int, dnsName string, dnsType uint16) (*dns.Msg, error) {
+type clientIP struct {
+	ip string
+	Opts *dns.OPT
+}
+
+func NewClientIP(ip string) (cip *clientIP) {
+	cip = new(clientIP)
+	cip.ip = ip
+	cip.Opts = &dns.OPT{}
+	if !cip.IsEmpty() {
+		subnet := &dns.EDNS0_SUBNET{
+			Code:          dns.EDNS0SUBNET,
+			Address:       net.ParseIP(ip),
+			Family:        1, // IP4
```

Comment:
where are the families id  defined?

Reply:
https://github.com/miekg/dns/blob/master/edns.go#L255